README.md

```diff
@@ -1,16 +1,40 @@
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fopenebs%2Frawfile-localpv.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fopenebs%2Frawfile-localpv?ref=badge_shield)
 
-RawFilePV
-===
+## RawFile-LocalPV
+The RawFile-LocalPV OpenEBS Data-Engine is a simialr but more flexible, yet more complex.. derication of the LocalPV-Hostpath Data-Engine. <BR>
```

Comment:
Typo - derivation
Remove double full-stop

Reply:
Fixed this.